operations/host_spawn.go

```diff
@@ -399,7 +399,7 @@ func hostConfigure() cli.Command {
 				return errors.Wrap(err, "setting up legacy Evergreen client")
 			}
 
-			projectRef, err := ac.GetProjectRef(project)
+			projectRef, err := ac.GetProjectWithWorkstationConfig(project)
```

Comment:
It seems like changing this will still break existing CLIs without some transition period. If they don't upgrade their CLI immediately and they run `evergreen host configure`, it won't work anymore.

Are users supposed to have a grace period to upgrade, or are they expected to upgrade their CLIs immediately for the command to work? If they're expected to have a grace period, we have to leave the existing `/ref` route as-is and make new ones.

Reply:
removed it and put it all in /ref